source/whats-new.txt

```diff
@@ -26,6 +27,52 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.11:
+
+What's New in 4.11
+------------------
+
+The 4.11 {+driver-short+} release includes **recursive schema support**
+and provides type safety for dot notation queries up to a depth of
```

Comment:
Issue: 
I think "recursive schema" is jargon that we should define. I referred to our existing docs on known limitations involving recursive types (https://www.mongodb.com/docs/drivers/node/current/fundamentals/typescript/#known-limitations)
and Googled, but I wasn't able to find the term recursive schema. This made it difficult for me to understand whether the conjunction between "recursive schema support" and "provides type safety for dot notation queries..." was intended, or whether it was a definition.

Suggestion:
I think, but am not certain, this is referring to "recursive types", but also that the recursion applies to creating types that correspond to collections that are recursive in nature. It could be worthwhile reaching out to the Node.js DBX team to ensure you can provide an accurate definition.

Also, depending on whether they're related, the conjunction could be replaced by a separate sentence that explains the term or introduces the second item (whichever applies here).

Reply:
Followed up and confirmed that recursive schema support refers to support for recursive types as the schema used in the `Collection` class